package mvp;

message OrbitalImageFileDescriptor {
  required string camera_path = 1;
  required string image_path = 2;
}
